lib/hanami/logger.rb

```diff
@@ -145,7 +149,7 @@ def call(severity, time, _progname, msg)
         _format({
           app:      @application_name,
           severity: severity,
-          time:     time.utc
+          time:     time.utc.strftime(TIME_RFC3339_FORMAT)
```

Comment:
@AlfonsoUceda I think we should pass just `time` to the formatters and let it to decide which timezone and format to use.

---

**Default formatter:** We said that default formatter is designed for human readability during the development phase.

While UTC is a timezone that it's a standard in production, it's not human friendly if you live far away from it and you're developing your project.

RFC 3339 is a standard for logging parsers, but we want to give up with parsing for this formatter.

IMO the default Ruby formatting for `Time` is enough, and it safes a few CPU cycles as we don't need to use `#strftime`.

```ruby
irb(main):001:0> now = Time.now
=> 2017-01-14 16:14:56 +010 # More readable, no CPU waste
irb(main):002:0> now.strftime("%FT%T%:z")
=> 2017-01-14T16:14:56+01:00 # Less readable, more CPU cycles
```

---

**JSON formatter:** This is for production (convert to UTC), and it must be parseable. In this case it makes sense to format the timestamp. I'm only wondering if in this case we should use ISO 8601, which is the de-facto standard format for JSON.

---

**Custom formatter:** The win of passing the `time` object as it is, is to give the freedom to developers to leave the current timezone or convert it to UTC. Also, they can decide their own time format.

---

WDYT?